substrate/pkg/controller/substrate/infrastructure/subnets.go

```diff
@@ -97,12 +97,22 @@ func (s *Subnets) ensureSubnet(ctx context.Context, substrate *v1alpha1.Substrat
 	if len(describeSubnetsOutput.Subnets) > 0 {
 		logging.FromContext(ctx).Infof("Found subnet %s", aws.StringValue(name))
 		return describeSubnetsOutput.Subnets[0], nil
+
+	}
+	// these tags are required by ELB controller to discover these subnets to configure ELB
+	tags := []*ec2.Tag{
+		{Key: aws.String("kubernetes.io/cluster/" + substrate.Name), Value: aws.String("shared")},
```

Comment:
IIUC this is no longer necessary.

Reply:
Yeah Thanks, I missed the last line `The cluster tag is not required in v2.1.2 and newer releases.` here -https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/deploy/subnet_discovery/